modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -1020,15 +1020,23 @@ const FDSEntries = ({
 						data: {
 							id: 'view',
 						},
-						icon: 'view',
-						label: Liferay.Language.get('view'),
+						icon: 'pencil',
+						label: Liferay.Language.get('edit'),
```

Comment:
To be consistent in code, can we change all `view*` in file into `*edit`:
- `onViewClick` -> `onEditClick`
- `getViewURL` -> `getEditURL`
- `id: 'view'` -> `id: 'edit'`, along with `actionId: 'edit'`,